pyproject.toml

```diff
@@ -57,8 +57,13 @@ python-dotenv = ">=0.20,<0.22"
 typing-extensions = "^4.2.0"
 simplejson = "^3.17.6"
 jsonschema = "^4.16.0"
+pyarrow = "^10.0.1"
```

Comment:
@jamielxcarter I would like to keep `pyarrow` and `numpy` as optional dependencies. Is it possible to add them to an `extra` array?

Reply:
Yeah, I'll change them back to being optional